### PR TITLE
FLAG-74: Add java 17 support

### PR DESCRIPTION
**Pull request by ManojLL**
Issue work on [FLAG-74](https://openmrs.atlassian.net/browse/FLAG-74)

[FLAG-74]: https://openmrs.atlassian.net/browse/FLAG-74?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -222,8 +222,28 @@
...
+			<id>Java 17</id>
+			<activation>
+				<jdk>17</jdk>
+			</activation>
+			<build>
+				<pluginManagement>
+					<plugins>
+						<plugin>
+							<groupId>org.apache.maven.plugins</groupId>
+							<artifactId>maven-compiler-plugin</artifactId>
```
Comment (wikumChamith) [inline pom.xml:237]: Can't we compile and run with Java 17 without adding this?

Comment (ManojLL) [reply pom.xml]: it is working without adding this part

**Comment by wikumChamith** [comment]
I don't think we need this PR if we can build and run the current module using Java 17 without any code changes. Test the with both O2 and O3 snapshots and give us an update.

**Comment by ManojLL** [comment]
This module build in java 17 without any code changes.

**Pull request closed without merging**